config_proc_macro/Cargo.toml

```diff
@@ -19,5 +19,5 @@ syn = { version = "1.0", features = ["full", "visit"] }
 serde = { version = "1.0", features = ["derive"] }
 
 [features]
-default = []
 debug-with-rustfmt = []
+default = []
```

Comment:
Not talked in style guide, but maybe unexpected

Reply:
It is mentioned in the style guide -- recommendation is to sort alphabetically minus the first few keys in [package]